sound/soc/intel/boards/Kconfig

```diff
@@ -656,6 +656,8 @@ config SND_SOC_INTEL_SOUNDWIRE_SOF_MACH
 	depends on MFD_INTEL_LPSS || COMPILE_TEST
 	depends on SND_SOC_INTEL_USER_FRIENDLY_LONG_NAMES || COMPILE_TEST
 	depends on SOUNDWIRE
+	select PINCTRL_CS42L43
+	select SPI_CS42L43
```

Comment:
maybe move down to line 682, after all CS42L43 stuff

Reply:
> maybe move down to line 682, after all CS42L43 stuff

Updated